dna/models/dag_lstm_regression_model.py

```diff
@@ -0,0 +1,55 @@
+from .lstm_model import LSTMModel
+from .torch_modules.dag_lstm_mlp import DAGLSTMMLP
+from dna.data import group_json_objects
+
+
+class DAGLSTMRegressionModel(LSTMModel):
+
+    def __init__(
+            self, activation_name: str, hidden_state_size: int, lstm_n_layers: int, dropout: float,
```

Comment:
Let's not do this double indentation in the function definitions. I will take care of it this time.

Reply:
Sorry my IDE does that automatically.